dev/scripts/release/analyze.sh

```
@@ -0,0 +1,284 @@
+      case "$conclusion" in
+        success)  echo "   ✅ $name" ;;
+        neutral)  echo "   ⚪ $name (skipped/neutral)" ;;
+        failure)  echo "   ❌ $name  ← FAILED" ;;
+        pending)  echo "   🔄 $name  (still running)" ;;
+        *)        echo "   ❓ $name ($conclusion)" ;;
+      esac
+    done
+    echo ""
+    if [[ -n "$CI_FAILED" ]]; then
+      echo "   ⚠️  FAILURES detected. Review before releasing:"
+      echo "$CI_FAILED" | sed 's/failure /   ❌ /'
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)